debian/postinst

```diff
@@ -7,14 +7,14 @@ ATE0
 AT
 AT#ENAEVMONI=0
 AT#ENAEVMONICFG=3,1,2
-AT#ENAEVMONI=1
```

Comment:
Is this an issue because we need to wait until all of the `SMSIN` and `GPIO3` events are set up before we enable it?

Reply:
![image](https://github.com/sabantoag/atinout/assets/27155636/ce620dc5-67b5-44c6-acdf-6385fc3b9d68)
Here is what I could find in the documentation. They don't reference explicitly disabling the service, but they do show all of their configuration commands being sent prior to enabling it (and it is disabled by default)

Reply:
So, there's more that shows them setting `AT#ENAEVMONI=1` after they set `AT#EVMONI="RING",1`?